docs/lesson/create.rst

```diff
@@ -205,7 +204,7 @@ Add Responses
    
    When a learner answers incorrectly, there is an option to get the learner to 'try again' or loop back and answer the question again. It isn't always a good idea to do this. Think about whether you are able to group the possible 'wrong' answers in such a way that you can provide detailed feedback to the learner. 
 
-   If you cannot do this, i.e., if the possible 'wrong' answers have come in through a text-field (so you have no way of knowing what the learner might type), or if there are lots of wrong answers to a question - it is better to send the learner to a separate branch to address their misconceptions. Looping them back to the question in this case might cause frustration as they will end up trying to guess what you want them to say.
+   If you cannot do this, i.e., if the possible 'wrong' answers have come in through a text-field (so you have no way of knowing what the learner might type), or if there are lots of wrong answers to a question—it is better to send the learner to a separate branch to address their misconceptions. Looping them back to the question in this case might cause frustration as they will end up trying to guess what you want them to say.
```

Comment:
One question about the em dashes -- should there be single spaces on each side of them? (Throughout.)

Reply:
This is a style choice, most style guides recommend no spaces before and after the em dash (Associated Press style guide is the exception).